src/content/docs/tutorial-errors/reduce-common-errors.mdx

```diff
@@ -0,0 +1,75 @@
+* Diagnose the root cause of an incident
+
+## Route errors to the correct teams [#route]
+
+To send errors to other teams, you need to return to the **Errors** summary page. If you don't own a component or functionality, you can use this page to send grouped errors to the correct teams.  
+
+    <img
+        title="Overview errors affecting your services"
+        alt="A screenshot showing an app with many errors"
+        src={apmTriageError}
+    />  
+
+In the previous tutorial docs, we covered how to quickly respond to a critical error. But even without a service outage, we recommend that you return to the **Errors** page for at least 10 minutes on a weekly cadence. 
+
+* Imagine a patch that was deployed 48 hours ago on a service. During your weekly triage, you notice a new error has appeared in your errors inbox. An alert hasn't fired because the error occurence hasn't met your alert threshold, but you do notice an increase in error incidences. 
+* You route this error to the correct team even though there's a chance it's an expected error.
+* Because the deployment was recently introduced, the owning team still has context to evaluate the scope of the problem. In short, the person who made the deployment is still familiar enough to reproduce the problem, something that might not be possible in several months time. 
+
+When you spend time maintaining your errors, you can more easily keep track of what's actually expected or what's anomalous. You become more independent when you're proactive instead of reactive to external notifications. 
+
+If that same team responds that that error is expected, you can then close out these errors by selecting **Ignore**. We recommend resolving expected errors as much as you can, however. Ignoring an error group doesn't prevent New Relic from reporting the error in the future, which contributes to your data ingest.
+
+## Identify the root cause [#root]
+
+Whether you're reducing common errors or reacting to a critical outage, you're following data that leads you to the direct cause of an error occurrence. You may have fixed the leaking pipe that flooded your yard, but you haven't yet figured out the why? behind the leaking pipe. 
+
+Similar to a leaking pipe, when you're solving a problem that affects the end user, you aren't solving the root cause of an incident. The service that's not receiving API calls is a direct cause to a component failing, but the root of the problem can usually be found elsewhere. It could be poor planning for a new release or a lack of communication between two separate teams. These root causes won't be captured by an observability tool. Rather, they're uncovered in retrospectives and meetings between the people creating and improving a system.
+
+Whatever the cause, until you've used incidents to discuss root causes, you'll continue to receive the same kinds of errors. Your app won't perform the way you want it to and the end user will have a poor experience. 
```

Comment:
Can this tone be more positive about how it will be if they do start being more proactive and doing more retros to prevent future issues?